template/.stylelintrc

```
@@ -1,9 +1,306 @@
 {
-  "extends": "stylelint-config-recommended",
+  "extends": "stylelint-config-idiomatic-order",
```

Are most of the added rules copied from `stylelint-config-recommended`? If so, can we extend multiple configs to reduce the number of rules we add?

```suggestion
  "extends": ["stylelint-config-recommended", "stylelint-config-idiomatic-order"],
```

From the [docs](https://stylelint.io/user-guide/configure#extends):
>You can extend an array of existing configurations, with each item in the array taking precedence over the previous item (so the second item overrides rules in the first, the third item overrides rules in the first and the second, and so on, the last item overrides everything else).

---

Ah, nice, I didn't know you could extend multiple configs. I'll make an update for that!